about.html

```diff
@@ -12,62 +12,79 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="container">
+    <header class="head-contain">
+      <div class="logo">
+        <img src="img/lambda-black.png" class="lambda-logo" alt=""/>
+      </div>
+      <nav>
```

Comment:
At least for this project, you can target the nav a and style it without using a class for each anchor tag.